app/views/teams/show.html.erb

```diff
@@ -30,7 +30,7 @@
   <h2>Game rounds</h2>
   <% if @team.rounds.present? %>
     <div class="row mt-3">
-      <% @team.rounds.each do |round| %>
+      <% @team.rounds.order(:created_at).each do |round| %>
```

Comment:
Do we need to order by `:created_at` explicitly? Not really sure.

Reply:
Yes. It was changing the order after updating some entries. I guess the default order uses the `updated_at` field.